data.md

```diff
@@ -6,43 +6,49 @@ Data imported for users (including trainers)
 --------------------------------------------
 
 - "siRNAi-HeLa" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/siRNAi-HeLa/
+   * https://downloads.openmicroscopy.org/images/DV/siRNAi-HeLa/
 - "Cajal-bodies" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/alexia/cajal-bodies
+   * https://downloads.openmicroscopy.org/images/DV/alexia/cajal-bodies
 - "CellProfiler images" dataset:
-   * http://downloads.openmicroscopy.org/images/TIFF/cellprofiler/fruit-fly-cells
+   * https://downloads.openmicroscopy.org/images/TIFF/cellprofiler/fruit-fly-cells
 - "Condensation" dataset:
-   * http://downloads.openmicroscopy.org/images/TIFF/condensation/
+   * https://downloads.openmicroscopy.org/images/TIFF/condensation/
 - "Fiji Batch" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_10_R3D_D3D.dv 
-   * http://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_11_R3D_D3D.dv
+   * https://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_10_R3D_D3D.dv 
+   * https://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_11_R3D_D3D.dv
+- "FRAP" dataset:
+   * https://downloads.openmicroscopy.org/images/DV/will/FRAP
+- "Kymograph" dataset:
+   * http://jcb-dataviewer.rupress.org/jcb/browse/4573/12710/ Image identified by letter F
 - "Multi-Z timelapse" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/iain/438CTR/
+   * https://downloads.openmicroscopy.org/images/DV/iain/438CTR/
 - "PTRE" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/elena/P-TRE/
+   * https://downloads.openmicroscopy.org/images/DV/elena/P-TRE/
 - "SVS" dataset:
-   * http://downloads.openmicroscopy.org/images/SVS/
+   * https://downloads.openmicroscopy.org/images/SVS/
 - "U2OS_Eg5_PH3_005_R3D.dv" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/U2OS/
+   * https://downloads.openmicroscopy.org/images/DV/U2OS/
 - "Lionheart MRI DICOM" dataset:
-    * Lionheart, W. R. B. (2015). An MRI DICOM data set of the head of a normal male human aged 52 [Data set]. Zenodo. http://doi.org/10.5281/zenodo.16956
+    * Lionheart, W. R. B. (2015). An MRI DICOM data set of the head of a normal male human aged 52 [Data set]. Zenodo. https://doi.org/10.5281/zenodo.16956
+- "artificial-trackmate" dataset:
+   * http://fiji.sc/samples/FakeTracks.tif
 
 
 Data imported for trainers only
 -------------------------------
 
 - "INMAC384-DAPI-CM-eGFP_59223_1.xdce" plate:
-   * http://downloads.openmicroscopy.org/images/HCS/INCELL2000/INMAC384-DAPI-CM-eGFP_59223_1/
+   * https://downloads.openmicroscopy.org/images/HCS/INCELL2000/INMAC384-DAPI-CM-eGFP_59223_1/
 - "chromatine-condensation" dataset:
-   * http://downloads.openmicroscopy.org/images/TIFF/condensation/
+   * https://downloads.openmicroscopy.org/images/TIFF/condensation/
```

Comment:
Everybody has this one. Move to upper section

Reply:
Sorry, this is highlighted correctly. This ``chromatine-condensation`` is called ``condensation`` and everybody has it.